packages/langchain/README.md

```diff
@@ -0,0 +1,122 @@
+# @sap-ai-sdk/langchain
+
+This package contains Langchain compliant models, based on the @sap-ai-sdk clients.
+
+### Installation
+
+```
+$ npm install @sap-ai-sdk/langchain
+
+$ npm install @langchain/openai // if you want to use OpenAI models
+```
+
+## Pre-requisites
+
+- [Enable the AI Core service in BTP](https://help.sap.com/docs/sap-ai-core/sap-ai-core-service-guide/initial-setup).
+- Project configured with Node.js v20 or higher and native ESM support enabled.
+- For testing your application locally:
+  - Download a service key for your AI Core service instance.
+  - Create a `.env` file in the sample-code directory.
```

Comment:
[q] How is that related to "testing your application locally"? Please note, that those are the instruction on npm. Seems a bit confusing to me and rather related to our sample code / E2E tests.

Reply:
I figured you have to follow the same steps when using this library and then testing your app.
I can see how this might be redundant though, as this could then be said for any library ever. 🤷